### PR TITLE
Add gitignore and example.env

### PR DESCRIPTION
**Pull request by WindShiba**

### CODE REVIEW

File: crewinterface/settings.py [hunk 1]
```diff
@@ -20,10 +25,10 @@
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-5*-kc^mwgvy)2g(8@lj$q6=ulwbj2b7!^8vyk28nvgaqmmmsp5"
+SECRET_KEY = os.getenv('Secret_key')
```
Comment (oddstr13) [inline crewinterface/settings.py]:
```suggestion
SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
```

File: crewinterface/settings.py [hunk 2]
```diff
@@ -20,10 +25,10 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('Debug')
```
Comment (oddstr13) [inline crewinterface/settings.py]:
```suggestion
DEBUG = os.getenv('DJANGO_DEBUG')
```

File: example.env [hunk 3]
```diff
@@ -0,0 +1,2 @@
+DEBUG=Bool
```
Comment (oddstr13) [inline example.env]:
```suggestion
DJANGO_DEBUG=True
```

File: example.env [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+SECRET_KEY="foo_Django_secret_key"
```
Comment (oddstr13) [inline example.env]:
```suggestion
DJANGO_SECRET_KEY="foo_Django_secret_key"
```

**Review by oddstr13 — CHANGES_REQUESTED** [review]

File: example.env (hunk 4 above)
Comment (WindShiba) [reply example.env]: yeee. we min max

**Review by oddstr13 — APPROVED** [review]

**Pull request merged**